code/processes/tickerlogreplay.q

```diff
@@ -187,6 +194,13 @@ finishreplay:{[h;p;td]
  .save.postreplay[h;p];
  }
 
+// takes in log file directories made with segmented tickerplant
+expandstplogs:{[logdirectories] // always a list
+  $[`~tplogdir;
+    raze ` sv' logdirectories,/:key logdirectories:hsym first logdirectories;
+    `$raze pars,/:'string key each `$pars:string[logdirectories],'"/"]
```

Comment:
This operation can be done without strings:
```suggestion
    ` sv'raze ld,/:'pars:key each ld:hsym logdirectories]
```

Reply:
don't think you need the `pars` variable in the suggested change